mlrun/config.py

```diff
@@ -388,6 +388,9 @@
         "store_prefixes": {
             "default": "v3io:///users/pipelines/{project}/model-endpoints/{kind}",
             "user_space": "v3io:///projects/{project}/model-endpoints/{kind}",
+            "offline": "projects/{project}/model-endpoints/{kind}",
+            "stream": "",
```

Comment:
If you look at `get_model_monitoring_file_target_path()` you'll see that if the `stream` path is empty or does not exist, then we apply the `store_prefix.default` value with the project name and `kind=stream`. Therefore I think it's not necessary to set the default v3io value for both of these stores because the default `stream` is actually retrieved from `store_prefix.default`:
```
        "store_prefixes": {
            "default": "v3io:///users/pipelines/{project}/model-endpoints/{kind}",
            "user_space": "v3io:///projects/{project}/model-endpoints/{kind}",
            "stream": "v3io:///users/pipelines/{project}/model-endpoints/stream",
        },
```
I can also remove the `"stream": "",` but I thought it will more clear for the user to understand where can modify the stream value path in the configuration (though it's not the recommended approach anyway).